schulze_condorcet/__init__.py

```diff
@@ -1,3 +1,4 @@
-from .schulze_condorcet import schulze_evaluate, Candidate, Vote
+from .schulze_condorcet import (
+    detailed_result, pairwise_preference, schulze_evaluate, Candidate, Vote)
 
-__all__ = ['schulze_evaluate', 'Candidate', 'Vote']
+__all__ = ['detailed_result', 'pairwise_preference', 'schulze_evaluate', 'Candidate', 'Vote']
```

Comment:
Why do we export `pairwise_preference`? I don't see why a consumer should be interested in this.

Reply:
We want to expose this information inside the cdedb, since it is quite interesting to see how the preferences between two particular options. While not legally binding, it can still be helpful to understand the sentiments of the voters. This is especially helpful if we use ballots more as a polling device, which occasionally is done.

Furthermore, it eases confirming that a change of the Satzung has indeed reached 2/3 majority by considering the pairwise preference of the winner and the bar.

So yeah, I see some possible interest there. ^^
See also https://tracker.cde-ev.de/gitea/cdedb/cdedb2/issues/2087

Reply:
That being said, this might be more appropriate as a part of the detailed results.

Reply:
Hmm, I dont really see how this can fit nicely in the current structure of the detailed result. Moreover, the detailed result can be constructed rather easily from the overall result obtained by schulze_condorcet and the pairwise preference.

So, I see no clear reason why (and how) we should include the pairwise_preference in the (current) detailed_result structure.

Reply:
The detailed result currently contains a subset of the pairwise preference, does it not?

I am not that familiar with the specifics though, maybe it is indeed easier to just keep them separate.

Reply:
I have decided to left them separate. Now, we expose the schulze_evaluate, schulze_evaluate_detailed and pairwise_preference functions to the outside.